007-flash/flash.sh

```diff
@@ -59,6 +60,11 @@ function doFlash()
 
     echo "  Compressing combined FASTQ into $out at $(date)." >> $log
     gzip $outUncompressed
+
+    if [ -f $singletons ]
```

Comment:
I would add a comment before this to say what's going on and why the file may not be present.

Reply:
done